pkg/gitprovider/gitlab/gitlab_test.go

```diff
@@ -376,6 +410,9 @@ func TestMergePullRequest(t *testing.T) {
 			if tc.expectPR {
 				require.NotNil(t, pr)
 				require.Equal(t, tc.id, pr.Number)
+				if tc.name == "merge queued (merge train)" {
```

Comment:
This is going to be brittle if we change test names. I would add a new `expectQueued` field instead. This would be good as well so you can test the negative (that it _wasn't_ queued) in the other tests